x/crosschain/keeper/zeta_conversion_rate.go

```diff
@@ -24,7 +24,7 @@ func (k Keeper) ConvertGasToZeta(context context.Context, request *types.QueryCo
 		return nil, status.Error(codes.InvalidArgument, "invalid request: param chain")
 	}
 	gasLimit := math.NewUintFromString(request.GasLimit)
-	outTxGasFee := medianGasPrice.Mul(gasLimit)
+	outTxGasFee := medianGasPrice.Mul(gasLimit).MulUint64(2) //FIXME: parameterize this to sync with where this fee is charged
```

Comment:
Why is it multiplied by 2?
I think we should add a comment in the code to add context

Reply:
That's because we overpay gas price by 2x to accommodate short term gas price spike. 

Need to put this in a parameter though.

Reply:
yes, a parameter would be good to use in both places and to modify via proposal